src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslMinPortSpeedSpec.groovy

```diff
@@ -18,7 +19,7 @@ Sometimes an ISL have different port speed on its edges.
 In that case, we need to set ISL capacity and all bandwidth parameters according to minimal speed value.
 Eg. 10G on one side, and 1G on another side, the ISL should have a 1G capacity.""")
 class IslMinPortSpeedSpec extends HealthCheckSpecification {
-    @Tags([SMOKE, TOPOLOGY_DEPENDENT])
+    @Tags([SMOKE, HARDWARE])
```

Comment:
Is it only for the HARDWARE env?

Reply:
OVS links don't support bandwidth configuration.